tools/csv-merger/csv-merger.go

```diff
@@ -674,6 +670,14 @@ func getUXBackendServerDeployment() appsv1.DeploymentSpec {
 								Name:  "TLS_ENABLED",
 								Value: os.Getenv("TLS_ENABLED"),
 							},
+							{
+								Name: util.OperatorNamespaceEnvVar,
```

Comment:
Use PodNamespaceEnvVar instead of OperatorNamespaceEnvVar

Reply:
ack